CHANGELOG.md

```diff
@@ -159,6 +159,8 @@ major version 0.y.z will maintain a stable public interface.
 * Clarify the gem installation instructions in the README (thanks
   [@nictrix])
 
+* Broken reference to the Getting Started guide
```

Comment:
This needs to be under the 0.6.0 release, which I realized was not yet in the file. 😆